Source/AwsCommonRuntimeKit/event-stream/EventStreamMessage.swift

```diff
@@ -54,36 +54,37 @@ extension EventStreamMessage {
         }
```

Comment:
above this diff, using `header.name.count` again

Reply:
Thanks, fixed.